src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,78 @@
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: 1024px;
+  margin: 0 auto;
```

Comment:
don't give margins and max-width to BEM-block. Give them to element class (page__cards)